devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy

```diff
@@ -17,7 +17,7 @@ import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 import physicalgraph.zigbee.zcl.DataType
 
 metadata {
-    definition (name: "SiHAS Multipurpose Sensor", namespace: "shinasys", author: "SHINA SYSTEM") {
+    definition (name: "SiHAS Multipurpose Sensor", namespace: "shinasys", author: "SHINA SYSTEM", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: true) {
```

Comment:
Setting `runLocally` and `executeLocally` to `true` will work as you expect. These should be removed.